logo.js

```diff
@@ -174,9 +174,25 @@ function LogoInterpreter(turtle, stream, savehook)
 
   // Returns a Promise that will resolve after yielding control to the
   // event loop.
```

Comment:
Can you update this comment? It now only guarantees a return to the event loop if it's been >= 20ms since the last yield.

Reply:
Maybe:

> Returns a Promise that will resolve as soon as possible while ensuring that control is yielded back to the event loop at least every 20ms.